readthedocs/oauth/notifications.py

```diff
@@ -59,7 +59,7 @@
     ),
     Message(
         id=MESSAGE_OAUTH_BUILD_STATUS_FAILURE,
-        header=_("{{provier_name}} build status report failed"),
+        header=_("{provider_name} build status report failed"),
```

Comment:
This is a django template :)

```suggestion
        header=_("{{ provider_name }} build status report failed"),
```

Reply:
```suggestion
        header=_("{{provider_name}} build status report failed"),
```